.github/ISSUE_TEMPLATE/usability.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Usability Improvement
+about: Make Bevy more fun to use!
+title: ''
+labels: C-Usability, S-Needs-Triage
+assignees: ''
+---
+
+## What tripped you up?
+
+Was something inconsistent, surprising or frustrating?
+Try to provide specific examples if you can.
+These could be code snippets, screenshots, a link to a bad error message in our source code, etc.
+
+If this is part of a broader problem, feel free to point out other areas where this needs to be fixed.
+
+## What would help you next time?
+
+What would your ideal solution look like?
+Talk about your dream API, the error messages you wish you had, or that one method you desperately need.
+
+## \[Optional\] Implementation strategy
```

Comment:
To me this section feels redundant with "What would help you next time? -> Talk about your dream api". I might rephrase it as:

```md
## What is your ideal solution?

Do we need better error messages? Are we missing a function?
If you have a specific implementation in mind, outline it here. 
```

The more "conditional" and structured we make a template, the more imposing it is.

I also like that "what would help you next time" and "what would your ideal solution look like" are compressed into a single sentence in the header. Just once less redundant thing for the user to read.